src/index.js

```diff
@@ -39,7 +39,6 @@ async function bundleMDX(
   // extract the frontmatter
   const {data: frontmatter} = matter(mdxSource)
 
-  //const dir = cwd ? cwd : path.join(process.cwd(), `__mdx_bundler_fake_dir__`)
```

Comment:
Yeah I only commented that out whilst I tested it. This is why you don't commit before you go to bed.